Controller/Adminhtml/Cart/Data.php

```diff
@@ -105,9 +105,8 @@ public function execute()
     {
         $startTime = $this->metricsClient->getCurrentTime();
         try {
-            $place_order_payload = $this->getRequest()->getParam('place_order_payload');
             // call the Bolt API
-            $boltpayOrder = $this->cartHelper->getBoltpayOrder(true, $place_order_payload);
+            $boltpayOrder = $this->cartHelper->getBoltpayOrder(true, '');
```

Comment:
Perhaps we should set `null` as the default value for the second parameter passed to \Helper\Cart::getBoltpayOrder ($placeOrderPayload) and not send this empty string here. Can you see if it does not break semething?

Reply:
> `Is it actually used anywhere? Maybe we can remove it completely?`

What do you mean? $placeOrderPayload in getBoltpayOrder ?

Reply:
And null as the default value for $placeOrderPayload would also works.

Reply:
Yes, $placeOrderPayload, is it used? If I remember correctly, it was develop with payment-only (default Magenbto checkout, frontend) support in mind which we don't do actually. And the back-office orders work without it, if I am not wrong.

Reply:
As far as I can see, $placeOrderPayload is used for front-end-payment-only and non-bolt-order (refer to PR https://github.com/BoltApp/bolt-magento2/pull/716  [code](https://github.com/BoltApp/bolt-magento2/pull/716/files#diff-4c4b44b5d24ce330e7833ae024f0071dR131)) , so I think we would better still keep it, and can consider to do some refactor on it (in another PR)

Reply:
And for the proposal to set null as default value, I think we can do it in another PR cause it seems to be irrelevant to the purpose of this PR